Server/src/models/AuthenticationModel.ts

```diff
@@ -116,10 +133,23 @@ export class AuthenticationModel {
         return userInfo[0];
     }
 
-    //TODO: Implement when doing password reset 
-    static async resetPassword(email: string) {
+    /**
+     * This will change user's resetToken attribute
+     * @param email - User's email
+     * @param token - User's resetToken
+     */
+    static async resetPassword(email: string, token: string) {
 
         let connection = getConnection();
+
+        await connection.manager
+            .createQueryBuilder(Accounts, 'accounts')
+            .update('accounts')
+            .set({ resetToken: token })
+            .where('accounts.email = :email', { email: email })
+            .execute()
```

Comment:
Does this let the user set the new token? Where is the user inputted password? Is it contained inside resettoken?

Reply:
No this is the first step, to assign a user a temp resetToken.. another method will be added for when the user actually changes their password